--- pnetlib-0.8.0/tools/Makefile.am.orig	2005-08-06 11:12:19.000000000 +0200
+++ pnetlib-0.8.0/tools/Makefile.am	2014-12-14 15:17:49.311055805 +0100
@@ -8,7 +8,7 @@
 	$(INSTALL_DATA) pinvoke.map $(DESTDIR)$(libdir)/cscc/lib/$(FRAMEWORK_VERSION)/pinvoke.map
 	if test "x$(INSTALL_AS_DEFAULT)" = "xtrue" ; then \
 		rm -f $(DESTDIR)$(libdir)/cscc/lib/pinvoke.map ; \
-		$(LN_S) $(DESTDIR)$(libdir)/cscc/lib/$(FRAMEWORK_VERSION)/pinvoke.map $(DESTDIR)$(libdir)/cscc/lib/pinvoke.map ; \
+		$(LN_S) $(FRAMEWORK_VERSION)/pinvoke.map $(DESTDIR)$(libdir)/cscc/lib/pinvoke.map ; \
 	fi
 
 uninstall-local:
